code/game/objects/objs.dm

```diff
@@ -230,6 +230,7 @@
 		buckled_mob.buckled = null
 		buckled_mob.anchored = initial(buckled_mob.anchored)
 		buckled_mob.update_canmove()
+		buckled_mob.density = initial(buckled_mob.density)
```

Comment:
This is _very_ bad. It means that every time you unbuckle somebody from, for example, a bed or a roller, he returns to being dense, even if he is lying down and by all rights should have no density. Make it somehing like `buckled_mob.update_density()`, which somebody probably should have made ages ago (it is already causing troubles with doublechairs under certain conditions).

Reply:
I made a variable to track it in resin nests, I am not up to writing another proc